Assets/Scripts/Battlefield/CreatureScripts/Ability.cs

```diff
@@ -1,14 +1,23 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using SwordAndBored.GameData.Abilities;
 
 namespace SwordAndBored.Battlefield.CreaturScripts {
-    [CreateAssetMenu(fileName = "Data", menuName = "Abilities/MeleeAttack", order = 1)]
-    public class MeleeAttackAbilitySO : AbstractAbilitySO
+    public class Ability : AbstractAbility
     {
+        /*
+        public Ability(CombatAbilities abilityStats)
+        {
+
+        }*/
+
+
+
```

Comment:
Can this be cleaned up?

Reply:
That's from battlefield, not part just part of earlier branch. Is resolved in alternate branch